src/services/api.ts

```diff
@@ -80,7 +103,7 @@ export class APIService {
   ): Promise<IApiCircle> => {
     const data = JSON.stringify(params);
     const { signature, hash } = await getSignature(data, this.provider);
-    const response = await axios.post('/circles', {
+    const response = await this.axios.post('/circles', {
```

Comment:
this can be changed to hit `/v2/circles` & use the auth token instead of a signature